### PR TITLE
[CDAP-6096] - Fixes cloning pipeline in CDAP.

### PR DESCRIPTION
**Pull request by ajainarayanan**
#### TL;DR -

  While cloning a pipeline, the UI was constantly failing when the user came from cdap (app status view). 
The reason for it was `ConfigStore` and `NodeConfigStore` were using same dispatchers. 

Renaming the dispatcher for `NodeConfigStore` fixed this problem.
#### Longer version -

  The detailed view and studio view of pipeline is sharing `NodeConfigStore` and this needs to be reset when switching between states. When the user is navigating from CDAP through pipeline detailed view to cloning a pipeline, resetting the store caused problems. The reason for it is we were using the same dispatcher for both `ConfigStore` and `NodeConfigStore`. It comes down to which store is first initialized and which dispatcher is initialized and which dispatcher is destroyed. When the  user clones navigating from cdap to hydrator to clone the wrong events (dispatcher) got destroyed and events that are supposed to be registered for initializing the store were missing. This is the reason why clone (which initializes the store with data) was failing.

No two stores can use the same dispatcher unless they are listening for the same events to be dispatched. In our case we should not have used the same dispatcher as those two stores are doing different things.

### CODE REVIEW

**Comment by elfenheart** [comment]
👍  LGTM
